specification/resources/resource-manager/Microsoft.Resources/stable/2024-11-01/resources.json

```diff
@@ -6845,6 +6853,18 @@
           }
         }
       }
+    },
+    "ValidationLevel": {
+      "type": "string",
+      "description": "The level of validation performed on the deployment. This value can be either Template, Provider, or ProviderWithoutWritePermissionChecks. At Template level, static analysis and validation of the template is performed. At Provider level, static analysis of the template is performed and resource declarations are sent to resource providers for semantic validation. At ProviderWithoutWritePermissionChecks level, static analysis of the template is performed and resource declarations are sent to resource providers for semantic validation without checking for write permissions.",
```

Comment:
Is it worth documenting the behavior of each value against the value itself, rather than having a large single description?

```json
"description": "The level of validation performed on the deployment.",
"enum": [
  "Template",
  "Provider",
  "ProviderWithoutWritePermissionChecks"
],
"type": "string",
"x-ms-enum": {
  "modelAsString": true,
  "name": "IncidentClassification",
  "values": [
    {
      "description": "Static analysis of the template is performed.",
      "value": "Template"
    },
    {
      "description": "Static analysis of the template is performed and resource declarations are sent to resource providers for semantic validation. Validates that the caller has RBAC write permissions on each resource.",
      "value": "Provider"
    },
    {
      "description": "Static analysis of the template is performed and resource declarations are sent to resource providers for semantic validation. Skips validating that the caller has RBAC write permissions on each resource.",
      "value": "ProviderWithoutWritePermissionChecks"
    }
  ]
}
```